dev-docs/bidders/bedigitech.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: bidder
+title: BEdigitech
+description: Prebid BEdigitech Bidder Adapter
+pbjs: true
+pbs: false
+biddercode: bedigitech
+gdpr_supported: false
+usp_supported: false
+media_types: banner, native
```

Comment:
The more meta data you add here, the better partners can vet you.

- safeframes_ok: true/false
- deals_supported: true/false
- floors_supported: true/false
- fpd_supported: true/false
- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- ortb_blocking_supported: true/partial/false
- userId: (list of supported vendors)

See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter